src/C++/FieldConvertors.h

```diff
@@ -640,7 +686,17 @@ struct UtcTimeOnlyConvertor
     else
       millis = 0;
 
-    return UtcTimeOnly( hour, min, sec, millis );
+    if( haveMicroseconds )
+    {
+      i += 3;
+      micros = (100000 * (value[i+1] - '0')
```

Comment:
Is this really a micro second? I think this is the nano second. Say you have 20171130-14:16:00.123456. I think this will give micros = 456000.